lib/class-wp-theme-json-resolver.php

```diff
@@ -140,8 +140,8 @@ private static function translate_presets( &$theme_json_structure, $domain = 'de
 						if ( empty( $item_to_translate[ $translatable_key ] ) ) {
 							continue;
 						}
-						// phpcs:ignore WordPress.WP.I18n.LowLevelTranslationFunction,WordPress.WP.I18n.NonSingularStringLiteralText,WordPress.WP.I18n.NonSingularStringLiteralDomain
-						$item_to_translate[ $translatable_key ] = translate( $item_to_translate[ $translatable_key ], $domain );
+						// phpcs:ignore WordPress.WP.I18n.LowLevelTranslationFunction,WordPress.WP.I18n.NonSingularStringLiteralText,WordPress.WP.I18n.NonSingularStringLiteralContext,WordPress.WP.I18n.NonSingularStringLiteralDomain
+						$item_to_translate[ $translatable_key ] = translate_with_gettext_context( $item_to_translate[ $translatable_key ], sprintf( 'theme %s %s', $path, $translatable_key ), $domain );
```

Comment:
I haven't try to execute it yet. I'm more flagging this issue that wasn't addressed in the original PR.

@jorgefilipecosta, any recommendations on how to compute the context name based on the path?